src/services/server-group/ServerGroupService.ts

```diff
@@ -163,6 +163,34 @@ export class ServerGroupService {
     return this.executeOp(serverGroup, "stop-servers");
   }
 
+  getChannelName(profile:string): ng.IPromise<string[]> {
+
+    let request: IDmrRequest = <IDmrRequest> {
+      address :['profile', profile, 'subsystem', 'datagrid-jgroups'],
+      'child-type': 'channel'
+    };
+    return this.dmrService.readChildName(request);
+  }
+
+  getCoordinator(server:IServerAddress, profile:string): ng.IPromise<IServerAddress> {
```

Comment:
We should really replace this with a call to JGroupsServie.getCoordinatorByServer as they are the same and then delete getChannelName(). It doesn't make sense to duplicate code